src/components/Image/Image.module.css

```diff
@@ -0,0 +1,3 @@
+.heading2 {
```

Comment:
We can change the name of the class to .image rather than .heading2 to give it a clearer name.